### PR TITLE
[MIG] stock_move_source_relocate_dynamic_routing: Migration to 14.0

### PR DESCRIPTION
**Pull request by dzungtran89**
Depends:
- stock_move_source_relocate: https://github.com/OCA/wms/pull/290

### CODE REVIEW

File: test-requirements.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
 freezegun
 odoo_test_helper
+odoo14-addon-stock_move_source_relocate @git+https://github.com/OCA/wms@refs/pull/290/head#subdirectory=setup/stock_move_source_relocate
```
Comment (simahawk) [inline test-requirements.txt]: please don't hide this into the migration commit. Add an extra commit like `Add TMP dependency` which makes prominent there's an extra commit to be discarded :wink:

**Review by simahawk — COMMENTED** [review]
LG

**Review by sebalix — COMMENTED** [review]
:+1:

**Comment by dzungtran89** [comment]
@sebalix There was a mistake of the migration commit message, I updated it

**Review by mmequignon — APPROVED** [review]

**Comment by dzungtran89** [comment]
fyi, I just removed the test deps commit

**Review by sebalix — APPROVED** [review]

**Pull request merged**